.expeditor/config.yml

```diff
@@ -80,6 +80,10 @@ subscriptions:
       # on this artifact's availability
       - trigger_pipeline:third-party-packages
 
+  # TODO is this a supported workload?
+  - workload: artifact_published:stable:habitat:{{version_constraint}}
```

Comment:
`artifact_published` is only for Artifactory artifacts. You can see how you can subscribe to Habitat promotions here: https://github.com/chef/automate/blob/master/.expeditor/config.yml#L172-L176